src/Files.App/Views/Properties/SecurityAdvancedPage.xaml

```diff
@@ -225,196 +228,198 @@
 							</Border>
 						</ListView.Header>
 						<ListView.ItemTemplate>
-							<DataTemplate x:DataType="perm:FileSystemAccessRuleForUI">
```

Comment:
To bind `IsSelected="{x:Bind IsSelected, Mode=TwoWay}"` in xaml code. This've been changed in the code-beind. No breaking changes.